cloudevents/formats/cbor-format.md

```diff
@@ -0,0 +1,199 @@
+# Avro Event Format for CloudEvents - Version 1.0.3-wip
+
+## Abstract
+
+The CBOR Format for CloudEvents defines how events attributes are expressed in
+the [CBOR Specification][cbor-spec].
```

Comment:
the ending does quite read right. Perhaps:

This specifications defines how to serialize Cloud Events in the [Concise Binary Object Representation(CBOR)][cbor-spec] data format.